pylama/lint/pylama_pydocstyle.py

```diff
@@ -1,6 +1,13 @@
 """pydocstyle support."""
 
-from pydocstyle import PEP257Checker
+THIRD_ARG = True
+try:
+    #: Import for pydocstyle 2.0.0 and newer
```

Comment:
This is a sphinx documentation comment. It is parsed by sphinx as documentation instead of only a comment to be ignored (by sphinx).